staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -62,7 +62,7 @@ var (
 		# Wait for the pod "busybox1" to contain the status condition of type "Ready"
 		kubectl wait --for=condition=Ready pod/busybox1
 
-		# The default value of status condition is true; you can set it to false
+		# The default value of status condition is true; you can other targets after an equal delimiter (compared after Unicode simple case folding):
```

Comment:
Word(s) missing here. “You can wait for other targets…”?

Reply:
I'm agnostic to that addition; seems orthogonal to whether/how we explain the presence of case folding.

Reply:
I was only commenting on the grammar. There's a verb missing from "you can other targets". Unless you're talking about [canning](https://en.wikipedia.org/wiki/Canning) the other targets :P

Reply:
🤦 , thanks.  Fixed with 8a129792565 -> b436cae85a9.